packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -349,9 +417,15 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   }
 
   get generatedAnswerClass() {
+    let collapsedClass = '';
+    if (this._exceedsMaximumHeight) {
+      collapsedClass = this.isExpanded
+        ? 'generated-answer__answer--expanded'
+        : 'generated-answer__answer--collapsed';
+    }
     return `generated-answer__answer ${
       this.isStreaming ? 'generated-answer__answer--streaming' : ''
-    }`;
+    } ${collapsedClass}`;
```

Comment:
```suggestion
    let collapsibleClass = '';
    if (this._exceedsMaximumHeight) {
      collapsibleClass = this.isExpanded
        ? 'generated-answer__answer--expanded'
        : 'generated-answer__answer--collapsed';
    }
    return `generated-answer__answer ${
      this.isStreaming ? 'generated-answer__answer--streaming' : ''
    } ${collapsibleClass}`;
```

maybe I would name this collapsibleClass since it covers both cases where it can be collapsed or expanded? (A bit less confusing)

Reply:
renamed it to `collapsedStateClass`